setup/README.md

```diff
@@ -16,7 +16,7 @@ A guide to setting up your Raspberry Pi
     - For keyboard layout configuration options see [raspi-config](../configuration/raspi-config.md).
 - [Power supply](../hardware/raspberrypi/power/README.md)
     - The Pi is powered by a USB Micro power supply (like most standard mobile phone chargers).
-    - You need a good-quality power supply that can supply at least 2A at 5V for the Model 3B and 3B+, or 700mA at 5V for the earlier, lower-powered Pi models. We recommend using the [official Raspberry Pi power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/), which is designed specifically for Raspberry Pi.
+    - You will need a good-quality power supply that can supply at least 2A at 5V for the Model 3B and 3B+, or 700mA at 5V for the earlier, lower-powered Pi models. We recommend using the [official Raspberry Pi power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/), which is designed specifically for Raspberry Pi.
```

Comment:
Out of curiosity, what's the difference?

Reply:
According to the Github-diff, he changed "You need a" to "You will need a"

Reply:
I mean in terms of grammar/meaning. It's true that you will need it in the future... but if you have a pi you also need it now. I guess the rest of the page uses 'will', so it keeps the tense consistent.

Reply:
Sorry about only adding one word. ill try to look for more in the meantime. I just started on git hub when this was posted. Please excuse that edit. as I will make some better in the future.

Reply:
ill close this as soon as possible